src/sidebar/search/GeocodingResult.tsx

```diff
@@ -34,19 +34,24 @@ const GeocodingEntry = ({
     isHighlighted: boolean
     onSelectHit: (hit: GeocodingHit) => void
 }) => {
+    const [wasCancelled, setWasCancelled] = useState(false)
+
     const className = isHighlighted
         ? styles.selectableGeocodingEntry + ' ' + styles.highlightedGeocodingEntry
         : styles.selectableGeocodingEntry
     return (
         <li className={styles.geocodingListItem}>
             <button
                 className={className}
-                onClick={() => {
-                    console.log('hit selected ' + entry.name)
-                    onSelectHit(entry)
-                }}
                 // prevent blur event for input textbox
-                onPointerDown={e => e.preventDefault()}
+                onPointerDown={e => {
+                    setWasCancelled(false)
+                    e.preventDefault()
```

Comment:
hm, you already do that ... strange.

Reply:
Okay added preventDefault on pointerup as well. Could you please try again? It works for me on Chrome, Firefox (both Desktop and emulated touch device) and iOS 14.6

Reply:
Unfortunately not. The same result. I also cleared the cache and restarted the app etc.

Maybe the underlying components get the click before the geocoding selection panel somehow?

Reply:
I have no clue what this means and why it works, but instead of your changes I use the original code and only added onTouchEnd, ie. 

```jsx
<button
    className={className}
    onClick={e => {
        onSelectHit(entry)
    }}
    onTouchEnd={e => {
        onSelectHit(entry)
        e.preventDefault()
    }}
    // prevent blur event for input textbox
    onPointerDown={e => e.preventDefault()}
>
```

and this works 🤔 (in firefox desktop, chrome desktop and firefox mobile)

Reply:
So, I tested this and also tried mutliple combinations of pointer events and touch and click event. I guess this works then. At least until someone tries to do thing with a stylus :-) 

This will use the click event for mouse interaction and the touch end for touch interaction. I would have found it more elegant to handle everything with pointer events. But I guess we'll leave it like this then.

Reply:
> I tested this and also tried mutliple combinations of pointer events and touch and click event. 

Thanks.

I would also keep your changes in AddressInput too as I find it more intuitive to remove focus (blur) as the first step before selecting an element.

> At least until someone tries to do thing with a stylus :-)

You mean these pens?

And yes, it is a bit unsatisfying that touch needs special care, but maybe we understand this better in the future.

I will merge (?)

Reply:
> Yes please.